examples/showcase/post-processing/index.html

```diff
@@ -0,0 +1,65 @@
+<html>
+
+<head>    
+     <script type="importmap">
+        {
+          "imports": {
+            "aframe": "../../dist/aframe-master.module.min.js",
+            "three": "../../../super-three-package/build/three.module.js",
+            "three/addons/": "../../../super-three-package/examples/jsm/",
+          }
+        }
+     </script>
+     <script type="module" src="./bloom.js"></script>;
+
+     <meta charset="utf-8">
+     <title>Post Processing Bloom A-Frame Demo</title>
+     <meta name="description" content="Post Processing Bloom Demo">
+
+</head>
+
+<body>
+
+    <a-scene         
+          reflection="directionalLight:#sun;"
+          shadow="type: pcfsoft; autoUpdate: true" 
+          background="color:black;"        
+          renderer="anisotropy:4; stencil:true; alpha:false; colorManagement:true; exposure:1.0;"
+          bloom="threshold: 1.0; strength: 0.6; radius: 1; exposure: 1.0">
+        
+        <a-entity id="pov-camera" position="0 1.6 2" rotation="3.323 -8.136 0" camera="" look-controls="" wasd-controls="" data-aframe-inspector-original-camera=""></a-entity>
```

Comment:
It seems the html was copied from an old version of the inspector.
You can remove `data-aframe-inspector-original-camera=""` ,
a bit weird also you have those values for rotation,
you can remove the `=""` here.